### ISSUE TITLE
NVDA users are not able to expand the list boxes and select value when navigated in browse mode(down arrow) and form mode shortcut

### ISSUE THREAD

**Issue by prsarnaik** [microsoft/fluentui#16504]
### Environment Information

**Package version(s)**: "office-ui-fabric-react": "^7.103.0",
**OS** : Win 10 2004 (OS Build 19041.388)
**Browser**: Chromium Edge Version 84.0.522.59
**Priority**: High
### Describe the issue:

Step 1: Enable screen reader NVDA and open the above URL in New Edge browser.

Step 4: Now navigate through screen in browse mode(down arrow) or form mode(f) to the list boxes and press enter to expand it and verify whether the list boxes are getting expanded.

Note: Only when navigated in tab key, users are able to expand the list boxes and select the value.

### Please provide a reproduction of the issue in a codepen:

Code Pen: https://codepen.io/pritam_sa/pen/LYRMydM

#### Actual behavior:

NVDA users are not able expand the list boxes and select their desired value, as the list boxes are not getting expanded when pressing enter, while navigating in browse mode(down arrow) or form mode shortcut(f).

#### Expected behavior:

NVDA users should be able expand the list boxes and select their desired value when pressing enter, while navigating in browse mode(down arrow) or form mode shortcut(f).

**Comment by gouttierre** [comment]
@prsarnaik - Thanks for the feedback.

@khmakoto  - Would you be able to determine if this is a regression or if this behavior is an issue? Also, if you can identity, how a NVDA user should be able expand the list boxes, while navigating in browse mode(down arrow).

**Comment by brunoprietog** [comment]
I am also an NVDA user and I can confirm that
